File: Python/60.py
import os
l=0
l1=0
l2=0
print("\n\nEste programa determine se uma palavra formada de tres letra forma um palindrome")
l=str(input("\n\nDigite a 1° letra dessa palavra : "))
l1=str(input("\n\nDigite a 2° letra dessa palavra : "))
l2=str(input("\n\nDigite a 3° letra dessa palavra : "))
if(l==l2):
    print("\n\nEsta palavra é um palindrome.")
else:
    print("\n\nEsta palavra não é um palindrome.")
print("\n\nObrigado por ultiliizar nosso programa.")
       

os.system("pause")

File: C/60.cpp
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main(void){
	setlocale(LC_ALL, "Portuguese");
	char a,b,c;
	printf("Esse programa determina se a palavra forma uma palíndrome ou não\n\n");
	
	printf("\n\nDigite a letra 1: ");
	scanf("%c",&a);
	printf("\n\nDigite a letra 2: ");
	scanf("%c",&b);
	printf("\n\nDigite a letra 3: ");
	scanf("%c",&c);
	
	if(a==c){
		printf("A palavra é uma palíndrome!");
	}else{
		printf("A palavra não é uma palíndrome!");
	}
	printf("\n\nObrigado por usar nosso sistema!\n\n");
	system("pause");
	return 0;
}


